airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.yaml

```diff
@@ -86,6 +86,18 @@ connectionSpecification:
       type: array
       items:
         type: integer
+    state_filter:
+      title: State Filter
+      description: Reflects the state of the Display Campaign stream as enabled, paused, or archived. If you do not populate this field, it will be ignored completely.
```

Comment:
this has to be consistent with spec.json

Reply:
thanks for catching this